Makefile

```diff
@@ -76,6 +76,27 @@ package-dashboard-rpm:
 
 	rm -rf ${PWD}/build
 
+### build rpm for apisix-openresty:
+.PHONY: package-apisix-openresty-centos7-rpm
+package-apisix-openresty-centos7-rpm:
+	chmod +x ./build-apisix-openresty-centos7.sh
+	./build-apisix-openresty-centos7.sh
+	fpm -f -s dir -t rpm \
+		-n apisix-openresty \
+		-a `uname -i` \
+		-v $(version) \
```

Comment:
Do I need to customize the `version` and `iteration` in a special way? I'm not sure

Reply:
We can pass them during build: https://github.com/api7/apisix-build-tools#parameters